index.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./main.css">
+    <title>Portfolio</title>
+</head>
+
+<body>
+    <header>
+        <nav>
+
+        </nav>
+        <nav class="nav-bar">
+            <div>Regiss Mukubiza</div>
+            <div><img src="./Images/Icon-Menu.svg" alt="menu"></div>
+        </nav>
+        <div class="header-title">
+            <h1>Hey there. I'm Regiss</h1>
+            <h3>I'am software developer</h3>
+            <p>I can help you build a product. feature or website look through some of my work and experience! if you
+                like what you see and have a project you need coded, don't hesitate to contact me.</p>
+            <div class="logo-plat">
+                <img src="./Images/github.svg" alt="github" />
+                <img src="./Images/linkedin.svg" alt="linkedin" />
+                <img src="./Images/Tert.svg" alt="Tert" />
+                <img src="./Images/twitter.svg" alt="twitter" />
+                <img src="./Images/Medium.svg" alt="Medium" />
```

Comment:
- [ ] Please wrap your social media links in list, You can use `<ul>` list. When we have similar elements in HTML it's  a good practice to wrap them in a list and it also makes sure our code is semantic.